packages/sdk-js/src/index.ts

```diff
@@ -19,13 +19,14 @@ import {
 	RevisionsHandler,
 } from './handlers';
 import { MemoryStore, BrowserStore } from './utils';
+import { createAxiosInstance } from './utils/create-axios';
 
 class DirectusSDK {
 	axios: AxiosInstance;
 	private authOptions: AuthOptions;
 
 	constructor(url: string, options?: { auth: Partial<AuthOptions> }) {
-		this.axios = axios.create({
+		this.axios = createAxiosInstance({
 			baseURL: url,
 			withCredentials: true,
 		});
```

Comment:
would you be open to optionally accepting either the `axios` config, or `axios`  instance?

this way a consumer has control over the http transport?

Reply:
Yeah totally! We already expose the axios instance itself through `.axios`, so I see no problem with allowing to configure that root axios instance (or passing it) on initialization of the SDK 👍🏻

Reply:
great, i'll add another commit to cover this

Reply:
good to review again @rijkvanzanten